src/doc/book/primitive-types.md

```diff
@@ -97,9 +97,10 @@ and `i64` is a signed, 64-bit integer.
 
 ## Variable sized types
 
-Rust also provides types whose size depends on the size of a pointer of the
-underlying machine. These types have ‘size’ as the category, and come in signed
-and unsigned varieties. This makes for two types: `isize` and `usize`.
+Rust also provides types whose particular size depends on the underlying machine
+architecture. Their range is sufficient to express the size of any collection, so
+these types have ‘size’ as the category. They come in signed and unsigned varieties
```

Comment:
I was talking about "They come". "Account for" sounds better from my point of view. :)